app/Http/Controllers/Three/PostsController.php

```diff
@@ -67,14 +67,22 @@ public function index(Request $request)
         $filters = $request->query('filter');
         $query = $this->filter($query, $filters, Post::$indexes);
 
-        // If user param is passed, return whether or not the user has liked the particular post.
-        if ($request->query('as_user')) {
-            $userId = $request->query('as_user');
-            $query = $query->with(['reactions' => function ($query) use ($userId) {
-                $query->where('northstar_id', '=', $userId);
+        // If a user made the request, return whether or not they liked each post.
+        if (auth()->check()) {
```

Comment:
should we document any of this stuff anywhere now without needing the `as_user` param?

Reply:
Yes! It looks like we use the same docs for v2 and v3, so I can split those out. 👍

Reply:
Awesome, thanks Dave!

Reply:
Done in cf960be!